src/components/Navbar/index.tsx

```diff
@@ -27,6 +29,7 @@ type Props = PublicProps & PropsFromState & ConnectedReduxProps;
 export class NavbarBase extends React.Component<Props> {
   static defaultProps = {
     _requestLogOut: requestLogOut,
+    reviewersHost: process.env.REACT_APP_REVIEWERS_HOST,
```

Comment:
`.env.common-local` needs to get `REACT_APP_REVIEWERS_HOST=https://reviewers.addons-dev.allizom.org`.

Currently, it's using the prod setting.